docs/guide/signing-data.md

```diff
@@ -287,7 +289,7 @@ If you’d like to read our JavaScript implementations of these methods, they ar
 
 Note that MetaMask supports signing transactions with Trezor and Ledger hardware wallets. These hardware wallets currently only support signing data using the `personal_sign` method. If you have trouble logging in to a website or dapp when using a Ledger or Trezor, the site may be requesting you sign data via an unsupported method, in which case we recommend using your standard MetaMask account.
 
-## A Brief History
+## <a name="history"></a> A Brief History
```

Comment:
```suggestion
## A Brief History
```
I don't think this is necessary, given that we generate anchor links. You should probably update the link above to `#a-brief-history`

Reply:
Ah, I wasn't quite sure how to do that. I'll try out the suggested anchor link instead.

Reply:
Worked like a charm!